tests/global_config_sno.py

```diff
@@ -34,6 +24,8 @@
             "default": True,
         }
     },
+    {HppCsiStorageClass.Name.HOSTPATH_CSI_BASIC: HPP_CAPABILITIES},
+    {HppCsiStorageClass.Name.HOSTPATH_CSI_PVC_BLOCK: HPP_CAPABILITIES},
```

Comment:
I would keep these in `global_config_sno_hpp.py`
So runs with `global_config_sno.py` and without storage-class-matrix param will correctly pick only LVMS.